Issue 38

iamalexk opened the pull request:
Fixed get_standings function from scraper.py and updates predictions.html to automatically update standings

skekre98 commented on app.py:
```diff
@@ -93,7 +93,10 @@ def authors():
-    return render_template("predictions.html", bracket=bracket)
+    west_standings = get_standings("west")
+    east_standings = get_standings("east")
+    print(east_standings)
```
Let's remove this print statement

skekre98 commented on modules/scraper.py:
```diff
@@ -133,19 +133,24 @@ def get_standings(conf):
     standings = []
-    for i,row in enumerate(table):
-        team = row.find("a")["title"]
+    # print(table[0])
```
Let's remove these extra comments

skekre98 reviewed (commented):
@iamalexk looks great! I have added a couple review comments. Can you also retrain the query classifier, it seems it was accidentally removed in one of your commits?

iamalexk commented:
@skekre98 I've fixed your comments, but what do you mean by retraining the query classifier?

skekre98 commented:
> @skekre98 I've fixed your comments, but what do you mean by retraining the query classifier?

@iamalexk if you look at the merge conflict, you'll see that git is complaining because `query_classifier.pkl` has been deleted. Not sure how this happened, but you can fix this by running `python query_classifier.py`

iamalexk commented:
@skekre98 Can you review my pull request. If query_classifier.pkl is still missing will you let me know. It looks to be in inference/models so I dont know why it would be causing a merge conflict. Thanks!

iamalexk commented:
@skekre98 Not sure what's going on with query_classifier.pkl. I added query_classifier.pkl in my commit but it still seems to have a conflicting file issue. Is there any way for you to merge just the app.py file and the modules/scraper.py files? I shouldn't have committed any of the changes I made to the other files I modified.

skekre98 commented on query_classifier.py:
```diff
@@ -6,7 +6,7 @@
 # ML packages
 from sklearn.svm import LinearSVC
 from sklearn.pipeline import Pipeline
-from sklearn.externals import joblib
```
Let's change this back to import `joblib` from `sklearn.externals`.

skekre98 commented on inference/inference_network.py:
```diff
@@ -1,5 +1,6 @@
 import pickle
 from sklearn.externals import joblib
+import joblib
```
Let's remove this extra joblib import.

skekre98 commented:
I think the issue is being created because you are using a different version of joblib and I'm assuming these versions encode their pickle files differently. Let's change this back to the original version of `joblib` that we were importing from `sklearn`.

skekre98 commented:
The brute force solution is to save the changes you've made to `app.py`, `scraper.py`, and `predictions.html` and `rebase` with the your fork. You can then re-add your changes and this will definitely resolve the merge conflict.

skekre98 commented:
> @skekre98 Not sure what's going on with query_classifier.pkl. I added query_classifier.pkl in my commit but it still seems to have a conflicting file issue. Is there any way for you to merge just the app.py file and the modules/scraper.py files? I shouldn't have committed any of the changes I made to the other files I modified.

@iamalexk Unfortunately I can only merge a pull request, not files, you will have to remove the unnecessary changes from your PR. Refer to this stack-overflow query about how to remove files from a PR: https://stackoverflow.com/questions/39459467/remove-a-modified-file-from-pull-request

Keep in mind, deleting a file and removing a file from a PR are two separate things.

iamalexk commented:
@skekre98 I have issued a new pull request called new-issue-38 that should hopefully fix our problems. Can we refer to that pull request from here?
